input/pagecontent/conformance-general.md

```diff
@@ -14,17 +14,17 @@ This IG currently only provides CapabilityStatements and documentation for "pull
 
 mCODE participants MUST meet the following requirements for conformance:
 
-1. [Identify mCODE patients](#identify-mcode-patients)
+1. [Identify in-scope patients](#identify-mcode-patients)
 1. [Follow conformance requirements for supported profiles](#follow-conformance-requirements-for-supported-profiles)
 1. [Populate and meaningfully process mCODE resources](#populate-and-meaningfully-process-mcode-resources)
 1. [Support querying mCODE-conforming resources](#support-querying-mcode-conforming-resources)
 1. [Publish a CapabilityStatement identifying supported profiles and operations](#publish-a-capabilitystatement-identifying-supported-profiles-and-operations)
 1. [Support US Core conformance requirements](#support-us-core-conformance-requirements)
 
 
-#### Identify mCODE Patients
+#### Identify In-Scope Patients
 
-To facilitate conformance testing, testing software must be able to determine which patients are "mCODE Patients" -- those in scope for mCODE. In general, all patients with confirmed cancer diagnoses SHOULD be covered by mCODE, but mCODE provides several ways to to identify the group of mCODE patients. See the [Identifying mCODE Patients](conformance-patients.html) page for details.
+To facilitate conformance testing, testing software must be able to determine which patients are "in-scope" -- Cancer patients who are in the scope of mCODE. In general, all patients with confirmed cancer diagnoses SHOULD be covered by mCODE, but mCODE provides several ways to to identify the group of mCODE patients. See the [Identifying mCODE Patients](conformance-patients.html) page for details.
```

Comment:
```suggestion
To facilitate conformance testing, testing software must be able to determine which patients are "in-scope" (meaning cancer patients who are in the scope of mCODE for conformance purposes). In general, all patients with confirmed cancer diagnoses SHOULD be covered by mCODE, but mCODE provides several ways to to identify this group of in-scope patients. See the [Identifying In-Scope Patients](conformance-patients.html) page for details.
```

Reply:
This defines "in scope" using the words "in scope". I would suggest:
To facilitate conformance testing, testing software must be able to determine which patients are "in-scope" (meaning cancer patients whose data is presented or exchanged with the intention of conforming to mCODE). In general, all patients with confirmed cancer diagnoses SHOULD be covered by mCODE, but mCODE provides several ways to to identify this group of in-scope patients. See the [Identifying In-Scope Patients](conformance-patients.html) page for details.